parts/k8s/kubernetesmastervars.t

```diff
@@ -166,7 +166,9 @@
     "vnetResourceGroupNameResourceSegmentIndex": 4,
 {{if IsHostedMaster}}
   {{if IsCustomVNET}}
+{{if .AgentPoolProfiles}}
```

Comment:
nit: indent this `{{if .AgentPoolProfiles}}` (and the `{{end}}`) so that it is 2 chars indented inside the `{{if IsCustomVNET}}` block.